domain-ee/ee-ep-merge-app/src/python_src/api.py

```diff
@@ -102,7 +101,16 @@ def validate_merge_request(merge_request: MergeEndProductsRequest):
 
 
 def start_job_state_machine(merge_job):
-    EpMergeMachine(merge_job).process()
+    EpMergeMachine(merge_job).start()
+
+
+def resume_job_state_machine(in_progress_job):
+    if in_progress_job.state == JobState.RUNNING_CANCEL_EP400_CLAIM:
+        EpMergeMachine(in_progress_job, Workflow.RESUME_CANCEL_EP400).start()
+    elif in_progress_job.state == JobState.RUNNING_ADD_CLAIM_NOTE_TO_EP400:
+        EpMergeMachine(in_progress_job, Workflow.RESUME_ADD_NOTE).start()
+    else:
+        EpMergeMachine(in_progress_job, Workflow.RESTART).start()
```

Comment:
I don't think this method will get more complicated, but in case it does, you could consider adding a little bit of cleverness to separate config (i.e. a dict mapping existing state to resumed state) from code.